### PR TITLE
replace with new images, add copy

### PR DESCRIPTION
**Pull request by BitcoinMechanic**

### CODE REVIEW

File: site/source/user-manual/reset-password.rst [hunk 1]
```diff
@@ -17,25 +17,31 @@ Reset Your Password
...
       :width: 60%
 
-#. Select your Embassy data drive
+#. Select your Embassy data drive.
 
-   .. figure:: /_static/images/setup/screen4-select_storage.jpg
+   .. figure:: /_static/images/setup/screen4-use_existing_drive_selection.png
       :width: 60%
 
-#. Enter and confirm a new password.  This time, save it somewhere safe!
```
Comment (kn0wmad) [inline site/source/user-manual/reset-password.rst:33]: Matt just added this, and it's probably fine - the user will have already set up a password previously, so they will understand the concept of the UI elements.  Also, they're hopefully self-explanatory

**Review by kn0wmad — CHANGES_REQUESTED** [review]
1 small change, otherwise gtg

File: site/source/user-manual/reset-password.rst (hunk 1 above)
Comment (BitcoinMechanic) [reply site/source/user-manual/reset-password.rst:33]: Yeah the implication that they forgot their password isn't necessarily true though...

Comment (BitcoinMechanic) [reply site/source/user-manual/reset-password.rst:33]: Changed it anyway

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**